.github/workflows/goCI.yml

```diff
@@ -19,6 +27,16 @@ jobs:
       os-dependencies: ${{ inputs.os-dependencies }}
     secrets: inherit
 
+  gitleaks:
+    if: inputs.gitleaks
+    uses: smallstep/workflows/.github/workflows/gitleaks.yml@max/gitleaks
+    secrets:
+      GITLEAKS_LICENSE_KEY: ${{ secrets.GITLEAKS_LICENSE_KEY }}
+
+  codeql:
+    if: inputs.codeql
+    uses: smallstep/workflows/.github/workflows/codeql-analysis.yml@max/gitleaks
+
```

Comment:
You're using a branch here. We should change this before merging.

Reply:
yep, mentioned that in the description as well. But still, appreciate the reminder. Will change once it's approved.